### PR TITLE
Removes backbone style

### PR DESCRIPTION
**Pull request by flovilmart**
- Use promises or await

### CODE REVIEW

**Comment by acinader** [comment]
uh, where does this show up?

**Comment by flovilmart** [comment]
This is the docs.parseplatform.org/js/guide :)

File: _includes/js/handling-errors.md [hunk 1]
```diff
@@ -5,39 +5,34 @@ Most Parse JavaScript functions report their success or failure using an object
...
-    alert("Everything went fine!");
-  },
-  error: function(model, error) {
-    // This will be called.
-    // error is an instance of Parse.Error with details about the error.
-    if (error.code === Parse.Error.OBJECT_NOT_FOUND) {
-      alert("Uh oh, we couldn't find the object!");
-    }
+const query = new Parse.Query(Note);
+query.get("aBcDeFgH").then((results) => {
```
Comment (acinader) [inline _includes/js/handling-errors.md:9]: maybe do a failed example with try catch and await?

File: _includes/js/promises.md [hunk 2]
```diff
@@ -222,18 +202,18 @@ query.find().then(function(results) {
...
 
 <pre><code class="javascript">
-var query = new Parse.Query("Comments");
+const query = new Parse.Query("Comments");
 query.equalTo("post", post);
 
 query.find().then(function(results) {
   // Collect one promise for each delete into an array.
-  var promises = [];
+  let promises = [];
```
Comment (acinader) [inline _includes/js/promises.md]: this could be `const`

**Review by acinader — APPROVED** [review]
wow.  nice job.

**Pull request merged**